Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM grubykarol/locust:0.13.5-python3.8-alpine3.11
```

Comment:
This should be updated according to our locust version. This config should be: 
FROM grubykarol/locust:0.11.0-python3.7-alpine3.9

Reply:
Fixed, Added another task to upgrade back to latest version in the container.